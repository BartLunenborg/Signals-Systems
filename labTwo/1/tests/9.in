10: [-50,29,-50,41,-23,-44,22,1,4,-19]
10: [-38,-46,40,13,16,-44,35,5,43,9]
