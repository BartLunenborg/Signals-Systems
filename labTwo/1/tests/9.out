19: [1900,1198,-1434,1252,-3435,6384,-3025,512,-5817,3154,846,-2002,-297,-2434,1531,-404,86,-781,-171]
